File: backend/semgrep_rules/tests/python/kisa_sw_46_private_array_return.py
class kisa_sw_46_private_array_return:
    def __init__(self):
        self._items = []

    def unsafe(self):
        # ruleid: kisa.sw46.python.private-list-return-review
        return self._items

    def safe(self):
        # ok: kisa.sw46.python.private-list-return-review
        return list(self._items)

File: backend/semgrep_rules/tests/javascript/kisa_sw_46_private_array_return.js
class kisa_sw_46_private_array_return {
  #items = [1, 2];

  unsafe() {
    // ruleid: kisa.sw46.javascript.private-array-return-review
    return this.#items;
  }

  safe() {
    // ok: kisa.sw46.javascript.private-array-return-review
    return [...this.#items];
  }
}
